packages/network/nodesList/manager/index.js

```diff
@@ -0,0 +1,29 @@
+import { compose } from 'redux';
+import { getPeers } from "@network/utilities/api";
+import withData from "@common/utilities/withData";
+import { DEFAULT_LIMIT } from '@views/configuration';
+import { withTranslation } from "react-i18next";
+import withLocalSort from "@common/utilities/withLocalSort";
+import { sortByVersion } from '../../utilities/helpers';
+import NodeList from "./manager";
+import { tokenMap } from '@token/configuration/tokens';
+
+export default compose(
+  withData({
```

Comment:
Same, Should not API calls be moved to managers?

Reply:
Similar to the other file. I think it's fine as it is. but please give more info if you think I have to change it.